commcare_connect/users/views.py

```diff
@@ -225,13 +226,10 @@ def test_func(self):
         return self.request.user.is_superuser
 
     def form_valid(self, form):
-        otp = get_user_otp(form.cleaned_data["phone_number"])
-        if otp is None:
-            messages.error(
-                self.request,
-                "Failed to fetch OTP. Please make sure the number is correct and"
-                " that the user has started their device seating process.",
-            )
+        try:
+            otp = get_user_otp(form.cleaned_data["phone_number"])
+        except ConnectIDClientError as e:
+            messages.error(self.request, str(e))
```

Comment:
I don't think we need to show the user the status code when there is an actual error, since there isn't anything they can do with that. It might be fine to just let it fail hard, but at worst a generic "something unexpected went wrong" message should be sufficient for the user.